sensor/kubernetes/sensor/sensor.go

```diff
@@ -191,12 +191,22 @@ func CreateSensor(cfg *CreateOptions) (*sensor.Sensor, error) {
 		components = append(components, k8sadmctrl.NewConfigMapSettingsPersister(cfg.k8sClient.Kubernetes(), admCtrlSettingsMgr, sensorNamespace))
 	}
 
-	// Local scanner can be started even if scanner-tls certs are available in the same namespace because
-	// it ignores secrets not owned by Sensor.
-	if securedClusterIsNotManagedManually(helmManagedConfig) && env.LocalImageScanningEnabled.BooleanSetting() {
+	if securedClusterIsNotManagedManually(helmManagedConfig) {
+		// Central capabilities are not available at this point (there's no connection to Central yet),
+		// so we'll start both Secured Cluster and Local Scanner TLS issuer Sensor components.
+		// Ideally we'd only start one or the other, depending on whether Central has
```

Comment:
They aren't respawned, on connection status changes there's a `Notify()` method in the Sensor component interface that gets called. We currently don't do anything in that method, and I'm not sure if we should.

Reply:
I'm not sure if you need to do anything in`Notify()` either. The idea is that if the connection to Central is disrupted, Sensor goes into offline mode, and reconnection to Central may happen later, and `Notify()` will let you know. In the meantime you want your component to not fail.

I suggest testing the robustness of your component against reconnection by putting it on a cluster and manually breaking the connection, for example with the chaos proxy we have under `sensor/common/chaos` if you're unsure.

Reply:
I tested and the component is able to recover if the Central connection is interrupted. To simulate this I changed the refresh interval from 6 months to every 1 minute and then breaking the connection to Central.

Because the component is unaware of Central connection status, the component still makes requests - which will timeout, but since it has a retry / backoff mechanism, it recovers automatically as soon as the connection is back.